sr_grasp_stability/src/sr_grasp_stability/tf2_computation.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+import rospy
+import tf2_ros
+
+
+class TfComputator:
+    """
+    Hand tf transformation computation
+    """
+
+    def __init__(self):
+
+        self.trans = {}
+        # self. rot = {}
+        self.tfBuffer = tf2_ros.Buffer()
+        self.listener = tf2_ros.TransformListener(self.tfBuffer)
+
+    def get_finger_tips(self, ref_frame):
+        """
+        Get fingertip transformations
+
+        () -> return (dict, dict)
+        """
```

Comment:
For any function that is documented please use the standard format. See example here 

https://github.com/shadow-robot/sr_interface/blob/indigo-devel/sr_robot_commander/src/sr_robot_commander/sr_robot_commander.py#L240